sprockets/mixins/http/__init__.py

```diff
@@ -152,13 +152,13 @@ def links(self):
         """
         if not self._responses:
             return None
+        links = []
         if 'Link' in self._responses[-1].headers:
-            links = []
             for l in headers.parse_link(self._responses[-1].headers['Link']):
                 link = {'target': l.target}
                 link.update({k: v for (k, v) in l.parameters})
                 links.append(link)
-            return links
+        return links
```

Comment:
Update the docstring as well ... still has `None` in the return type

Reply:
It does still return `None` elsewhere in the function.

Reply:
face-meet-palm